OrbitGl/AsyncTrack.cpp

```diff
@@ -51,6 +51,13 @@ void AsyncTrack::UpdateBoxHeight() {
   }
 }
 
+std::vector<std::shared_ptr<TimerChain>> AsyncTrack::GetAllChains() {
```

Comment:
There's a similar issue with the frame tracks, which also is using data that is already present in other tracks. Do we need a clear concept of "derived tracks" that are never serialized, but are computed on load?

Reply:
I added an explicit GetAllSerializableChains() method that the tracks can override, I think the solution is cleaner now. Essentially, a track now has control on exactly what timers should be serialized.

Reply:
Great, I was going to suggest the same!

Reply:
Yep, that makes it more clear. Thanks!